.github/workflows/issue-triage.yml

```diff
@@ -79,6 +79,8 @@ jobs:
 
   issue-accepted:
     runs-on: ubuntu-22.04
+    permissions:
+      issues: write
```

Comment:
sorry for one more thing. could you double check if that's still necessary when using github api to remove labels?

Reply:
I tried it in a private repo without the permissions first, and it would error with a 403.

It's possible that I could add `github-token: ${{ secrets.EXPO_BOT_GITHUB_TOKEN }}` and that would have permissions to remove labels, but I'm not sure know how I could verify that without merging the PR.

<img width="1119" alt="Screenshot 2023-10-13 at 15 32 49" src="https://github.com/expo/expo/assets/6534400/0e20dda9-a537-42a2-869f-d18506daabb3">